### PR TITLE
now exposing the correct file

### PR DESCRIPTION
**Pull request by hx-markterry**
#### What does this PR do? (please provide any background)

This work allows this module to be consumed correctly as now the main library is exported rather than the example code.

#### What tests does this PR have?

No additional tests

#### How can this be tested?

Check if you can include this code in another project and the correct object is available to you.

#### What gif best describes how you feel about this work?
giphy

---

- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

**Comment by jameshhx** [comment]
Taking Guru

File: README.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ An example script has been included, which demonstrates `snyk-report` being run
       console.log(output);
     });
 
+To run the example script use the following command:
+
+    npm run start
```
Comment (notABluesSinger) [inline README.md]: You don't need the `run` here... can just use `npm start`

**Comment by notABluesSinger** [comment]
Now that you've moved `index.js` into `script/example.js` you'll also need to update _Line 3_ from;
```
var report = require('./lib/report');
```
to
```
var report = require('../lib/report');
```

File: package.json [hunk 2]
```diff
@@ -1,13 +1,18 @@
 {
   "name": "snyk-report",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Assists in generating human readable snyk reports for CI integration",
+  "main": "lib/report.js",
   "scripts": {
-    "start": "node index.js",
+    "start": "node script/example.js",
     "test": "./node_modules/.bin/mocha test/unit",
```
Comment (notABluesSinger) [inline package.json]: We don't currently have any tests, so running `npm test` results in a failure, I know you haven't changed this as part of this PR, but could you update this, until we have some tests to run?

**Comment by notABluesSinger** [comment]
Sorry, taking slot 1

**Comment by notABluesSinger** [comment]
Final point, I've just noticed that the 1st checkbox on this PR points to a *contributing document* that doesn't exist. Do we have a standard document that should be added to this repo?

**Comment by hx-markterry** [comment]
Thanks for the feedback @notABluesSinger , i'm going to put a generic PR into the culture repo for a CONTRIBUTING file that we should point in the case of a missing project specific CONTRIBUTING file.

File: package.json (hunk 2 above)
Comment (jameshhx) [inline package.json]: I've a PR I'm putting together for tests - happy for it to be removed here, but will be added back in in the next PR.

Comment (hx-markterry) [inline package.json]: I've now removed the test script

**Comment by hx-markterry** [comment]
@notABluesSinger There is now a PR for the contributing fix: https://github.com/holidayextras/culture/pull/76

**Comment by notABluesSinger** [comment]
:shipit:

**Pull request merged**